.github/workflows/continuous-delivery.yaml

```diff
@@ -39,7 +39,13 @@ jobs:
           commit-assets: |
             ./dist
           node-module: true
+      - env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN_DOTTBOTT }}
+        id: tag
+        if: steps.released.outputs.released == 'true'
+        name: Tag
+        uses: Actions-R-Us/actions-tagger@v2
```

Comment:
Why can't we just use git for it? Looks like we could just use `persist-credentials: true`?

Reply:
As in tag with the git CLI?

Reply:
This is the semver regex that splits a version number into relevant capture groups:

`^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$`

Groups 1-3 are the major, minor and patch versions.
Group 4 is the prerelease tag.
Group 5 is the build metadata.

Our tag should look like this: `v{major}-{prerelease tag}`. How do I go about generating this string using sed? Can you lend a hand, I'm sort of drowning into the regex black hole here? 😬

Reply:
Ugh. Maybe we should have outputs with specific information? Is this exposed by Semantic?